src/frontend/utils.ts

```diff
@@ -21,59 +21,71 @@ export default class FrontendUtilities extends Utilities {
   }
 
   /**
-  * Sends a request to real BTT built in webserver with given data translated as GET query params
-  */
+   * Sends a request to real BTT built in webserver with given data translated as GET query params
+   */
   public async callBetterTouchTool(
-    action: string, 
+    action: string,
     data: Types.BttPayload,
     config: Types.AppConfig,
-    translate: boolean = true,
+    translate: boolean = true
   ): Promise<Types.CallResult> {
     if (this.isInBttWebView) {
+      if (config.sharedSecret) {
```

Comment:
Please add a comment above this line that says something like: `// handle secret used for thirdparty websites within web view`